tests/system/providers/google/cloud/vertex_ai/example_vertex_ai_auto_ml.py

```diff
@@ -0,0 +1,688 @@
+    )
+
+    create_forecast_dataset = CreateDatasetOperator(
+        task_id="forecast_dataset",
+        dataset=FORECAST_DATASET,
+        region=REGION,
+        project_id=PROJECT_ID,
+    )
+    forecast_dataset_id = create_forecast_dataset.output['dataset_id']
+
+    # [START how_to_cloud_vertex_ai_create_auto_ml_forecasting_training_job_operator]
+    create_auto_ml_forecasting_training_job = CreateAutoMLForecastingTrainingJobOperator(
+        task_id="auto_ml_forecasting_task",
+        display_name=f"auto-ml-forecasting-{ENV_ID}",
+        optimization_objective="minimize-rmse",
+        column_specs=COLUMN_SPECS,
+        # run params
+        dataset_id=forecast_dataset_id,
+        target_column=TEST_TARGET_COLUMN,
+        time_column=TEST_TIME_COLUMN,
+        time_series_identifier_column=TEST_TIME_SERIES_IDENTIFIER_COLUMN,
+        available_at_forecast_columns=[TEST_TIME_COLUMN],
+        unavailable_at_forecast_columns=[TEST_TARGET_COLUMN],
+        time_series_attribute_columns=["city", "zip_code", "county"],
+        forecast_horizon=30,
+        context_window=30,
+        data_granularity_unit="day",
+        data_granularity_count=1,
+        weight_column=None,
+        budget_milli_node_hours=1000,
+        model_display_name=f"auto-ml-forecasting-model-{ENV_ID}",
+        predefined_split_column_name=None,
+        region=REGION,
+        project_id=PROJECT_ID,
+    )
+    # [END how_to_cloud_vertex_ai_create_auto_ml_forecasting_training_job_operator]
+
+    # [START how_to_cloud_vertex_ai_delete_auto_ml_training_job_operator]
+    delete_auto_ml_forecasting_training_job = DeleteAutoMLTrainingJobOperator(
+        task_id="delete_auto_ml_forecasting_training_job",
+        training_pipeline_id=create_auto_ml_forecasting_training_job.output['training_id'],
+        region=REGION,
+        project_id=PROJECT_ID,
+    )
+    # [END how_to_cloud_vertex_ai_delete_auto_ml_training_job_operator]
+
+    delete_forecast_dataset = DeleteDatasetOperator(
+        task_id="delete_forecast_dataset",
+        dataset_id=forecast_dataset_id,
+        region=REGION,
+        project_id=PROJECT_ID,
+        trigger_rule=TriggerRule.ALL_DONE,
+    )
+    delete_bucket = GCSDeleteBucketOperator(
+        task_id="delete_bucket", bucket_name=FORECAST_GCS_BUCKET_NAME, trigger_rule=TriggerRule.ALL_DONE
+    )
+
+    clear_folder = BashOperator(
+        task_id="clear_forecast_folder",
+        bash_command="rm -r /forecast/*",
+    )
+
+    (
+        # TEST SETUP
+        create_bucket
+        >> unzip_file
+        >> upload_files
+        >> create_forecast_dataset
+        # TEST BODY
+        >> create_auto_ml_forecasting_training_job
+        # TEST TEARDOWN
+        >> delete_auto_ml_forecasting_training_job
+        >> delete_forecast_dataset
+        >> delete_bucket
+        >> clear_folder
+    )
+
+
+with models.DAG(
+    f"{DAG_ID}_image_training_job",
+    schedule_interval="@once",
```

Comment:
schedule_interval -> schedule for all DAGs

Reply:
Done